src/services/query.ts

```diff
@@ -106,3 +108,9 @@ export class QueryTermsService {
     });
   }
 }
+
+const alphabeticallyByPrefLabel = (a: Term, b: Term) => {
+  const prefLabelA = a.prefLabels[0]?.value ?? '';
+  const prefLabelB = b.prefLabels[0]?.value ?? '';
```

Comment:
`localCompare` must take a non-null string, so default to an empty string here.

Reply:
👍🏻